packages/devtools_app/lib/src/code_size/vm_treemap.dart

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
```

Comment:
add a TODO to delete this file once a new version of vm_snapshot_analysis is published. maybe even put it in a folder named `tmp` to make it clear that this is a temporary file

Reply:
Done